ui-tests/src/test/resources/features/customizations/jdbc-driver-extension.feature

```diff
@@ -0,0 +1,15 @@
+@jbdc-driver-import
+Feature: JDBC driver import
+
+  @check-connection
+  Scenario: Validate
+    Given allocate new "oracle12cR1" database for "Oracle12" connection
+    Given clean application state
+    Given "Camilla" logs into the Syndesis
+    Given import extensions from syndesis-extensions folder
+      | syndesis-library-jdbc-driver |
+
+    And she stays there for "60000" ms
```

Comment:
I'll try to apply isPodReady(..) then

Reply:
just one note. this is not so easy task. 
since pod reloading has several step. and isPodReady and ispodrunning are not useful here.
I played with it, but but I didn't find a way to make it fundamentally.
Anyway, I did it in some semi-fundamental way (see OpenShiftWaitUtils->waitForPodIsReady),